pub/refget-openapi.yaml

```diff
@@ -159,7 +159,8 @@ components:
             ga4gh:
               type: string
               description: >-
-                A ga4gh identifier used to identify the sequence. This is a base64url-encoded encoded
+                A ga4gh identifier used to identify the sequence. This is a SHA-512 
+                [base64 url encoded](https://datatracker.ietf.org/doc/html/rfc4648#section-5) 
```

Comment:
Thanks for the link, but the text was mostly better prior. “SHA-512” already appears later in the sentence; the name of the encoding defined in RFC4648 §5 is `base64url` (no space — this is unrelated to url-encoding, i.e. percent encoding, so best not to risk confusion with that).

Reply:
Good point. Changing this back